src/sentry/models/projectkey.py

```diff
@@ -51,6 +52,19 @@ def post_delete(self, instance, **kwargs):
             public_key=instance.public_key, trigger="projectkey.post_delete"
         )
 
+    def user(self):
+        """Return objects for the default use case"""
+        return self.get_queryset().filter(use_case=UseCase.USER.value)
+
+
+class UseCase(enum.Enum):
+    # A user-visible project key.
+    USER = "user"
+    # An internal project key for submitting aggregate function metrics.
+    PROFILING = "profiling"
+    # An internal project key for submitting escalating issues metrics.
+    ESCALATING_ISSUES = "escalating_issues"
```

Comment:
I think enum variants are documented like this in Python (I didn't find docs/PEP in some quick searches):
```suggestion
    USER = "user"
    """A user-visible project key."""
    PROFILING = "profiling"
    """An internal project key for submitting aggregate function metrics."""
    ESCALATING_ISSUES = "escalating_issues"
    """An internal project key for submitting escalating issues metrics."""
```

Reply:
Makes sense, updated.